### PR TITLE
Add rp_name setting

### LINKED ISSUE #37: Add rp_name to settings
Right now, `rp_name` needs to be overrided in some views for `django-fido` to work. It would be convenient to load `rp_name` from django settings, allowing to use `django-fido` views without further customization.

### PR DESCRIPTION
**Pull request by stinovlas**
Closes #37.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog #
 
+## Unreleased ##
+ * Add ``DJANGO_FIDO_RP_NAME`` setting.
```
Comment (tpazderka) [inline CHANGELOG.md]: Should be noted that this might be a breaking change if `rp_name` was previously set as class attribute.

File: README.md [hunk 2]
```diff
@@ -28,6 +28,8 @@ Django-fido provides basic components for FIDO 2 authentication - model to store
           url(r'', include('django_fido.urls')),
        ]
 
+4. Set string variable `DJANGO_FIDO_RP_NAME`.
```
Comment (tpazderka) [inline README.md]: This is, strictly speaking, only necessary for `python-fido >= 0.15`. Might be worth mentioning.

Comment (stinovlas) [reply README.md]: I added a note.

File: CHANGELOG.md (hunk 1 above)
Comment (stinovlas) [reply CHANGELOG.md]: We discussed that it's actually not a breaking change. `rp_name` can still be overridden by setting a class attribute.

**Pull request merged**